README.md

```diff
@@ -14,7 +14,8 @@ I always wanted the super fast feedback that codesandbox provides in my local en
 * Super tiny (24kb) 👌
 * Run projects without npm install 💃
 * React fast refresh ❤️
-* Supports React, Vue 2, Vue 3, Angular 🔨
+* Supports React, Vue 2, Vue 3, Angular, Preact, Svelte 🔨
+* Save disk space 💾
```

Comment:
This sounds good, but I think we should add the node_modules point in the title. And the disk space can remain as is. 

Because the main feature of the bundler is not having to install node_modules which saves disk space and time. What do you think ?

Reply:
cool makes sense